### PR TITLE
dhcpv6: add IA_PD to request from advertise

### PR DESCRIPTION
**Pull request by mikma**
Add the IA_PD option to the request from the advertise if present.
Add WithIAPD() as modifier and test.

### CODE REVIEW

File: dhcpv6/dhcpv6message.go [hunk 1]
```diff
@@ -139,6 +139,11 @@ func NewRequestFromAdvertise(adv *Message, modifiers ...Modifier) (*Message, err
 		return nil, fmt.Errorf("IA_NA cannot be nil in ADVERTISE when building REQUEST")
 	}
 	req.AddOption(iaNa)
+	// add IA_PD
+	iaPd := adv.GetOneOption(OptionIAPD)
+	if iaPd != nil {
```
Comment (pmazzini) [inline dhcpv6/dhcpv6message.go]:
```suggestion
	if iaPd := adv.GetOneOption(OptionIAPD); iaPd != nil {
```

Comment (pmazzini) [inline dhcpv6/dhcpv6message.go]:
```suggestion
	if iaPd := adv.GetOneOption(OptionIAPD); iaPd != nil {
```

Comment (mikma) [reply dhcpv6/dhcpv6message.go]: Fixed

**Comment by pmazzini** [comment]
Unit test needs to be fixed after #343 was merged.

**Comment by pmazzini** [comment]
There are conflicts after merging #361.

**Pull request merged**